extra_data/reader.py

```diff
@@ -1449,6 +1449,16 @@ def open_run(
         files which don't have this flag, out-of-sequence train IDs are suspect.
         If True, it tries to include these trains.
     """
+    if data is Ellipsis or data == 'all':
```

Comment:
I'd lean towards just one way (`data='all'`) to indicate this - `data=...` is cool but a bit cryptic.

(I think we used `data='proc'` rather than `corrected=True` because we anticipated one day adding an option to combine the two :+1: )